server/tserver/src/test/java/org/apache/accumulo/tserver/util/CreateEmptyTest.java

```diff
@@ -0,0 +1,206 @@
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+public class CreateEmptyTest {
+  @TempDir
+  private static File tempDir;
+
+  private ServerContext context;
+
+  @BeforeEach
+  public void init() throws IOException {
+    ConfigurationCopy config = new ConfigurationCopy(DefaultConfiguration.getInstance());
+    config.set(Property.INSTANCE_VOLUMES.getKey(), "file:///");
+
+    context = mock(ServerContext.class);
+    expect(context.getCryptoFactory()).andReturn(new GenericCryptoServiceFactory()).anyTimes();
+    expect(context.getConfiguration()).andReturn(config).anyTimes();
+    expect(context.getHadoopConf()).andReturn(new Configuration()).anyTimes();
+    VolumeManager volumeManager = VolumeManagerImpl.get(config, new Configuration());
+    expect(context.getVolumeManager()).andReturn(volumeManager).anyTimes();
+    replay(context);
+  }
+
+  @AfterEach
+  public void verifyMock() {
+    verify(context);
+  }
+
+  @SuppressFBWarnings(value = "PATH_TRAVERSAL_IN", justification = "path provided by test")
+  @Test
+  public void exceptionOnFileExistsTest() throws Exception {
+    CreateEmpty createEmpty = new CreateEmpty();
+
+    String wal1 = genFilename(tempDir.getAbsolutePath() + "/empty", ".wal");
+    String rf1 = genFilename(tempDir.getAbsolutePath() + "/empty", ".rf");
+
+    // create the file so it exists
+    File f = new File(wal1);
+    assertTrue(f.createNewFile());
+
+    String[] walArgs = {"--type", "WAL", wal1};
+    CreateEmpty.Opts walOpts = new CreateEmpty.Opts();
+    walOpts.parseArgs("accumulo create-empty", walArgs);
+
+    assertThrows(IllegalArgumentException.class,
+        () -> createEmpty.createEmptyWal(walOpts, context));
+
+    // create the file so it exists
+    File f2 = new File(rf1);
+    assertTrue(f2.createNewFile());
+
+    String[] rfArgs = {"--type", "RF", rf1};
+    CreateEmpty.Opts rfOpts = new CreateEmpty.Opts();
+    rfOpts.parseArgs("accumulo create-empty", rfArgs);
+    assertThrows(IllegalArgumentException.class,
+        () -> createEmpty.createEmptyRFile(walOpts, context));
+  }
+
+  @Test
+  public void createRfileTest() throws Exception {
+    CreateEmpty createEmpty = new CreateEmpty();
+
+    String file1 = genFilename(tempDir.getAbsolutePath() + "/empty", ".rf");
+    String file2 = genFilename(tempDir.getAbsolutePath() + "/empty", ".rf");
+
+    String[] args = {"--type", "RF", file1, file2};
+    CreateEmpty.Opts opts = new CreateEmpty.Opts();
+    opts.parseArgs("accumulo create-empty", args);
+
+    createEmpty.createEmptyRFile(opts, context);
+    VolumeManager vm = context.getVolumeManager();
+    assertTrue(vm.exists(new Path(file1)));
+    assertTrue(vm.exists(new Path(file2)));
```

Comment:
Could attempt to read these files using the RFile APIs in the public API.  Could open the file and assert the number read is zero like the following.

```java
    try(var scanner = RFile.newScanner().from(file1).build()){
      assertEquals(0, scanner.stream().count());
    }
```

Reply:
Thanks!  Added read checks in ede1e8a803